src/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const API_BASE_URL = 'https://transaction.api.cx.metamask.io';
 export const CHAIN_IDS = {
```

Comment:
It's an awkward one to maintain but the `NetworkController` has a list of chain IDs, plus the `TransactionController`, plus the extension itself.

I wonder if it's worth using a single source of truth in future such as the `NetworkController`.

Reply:
That would be nice if all controllers and extension / mobile use one source of truth for chainIds. Will get it from the NetworkController.